pkg/block/index.go

```diff
@@ -328,6 +339,7 @@ func GatherIndexHealthStats(logger log.Logger, fn string, minTime, maxTime int64
 		if ooo > 0 {
 			stats.OutOfOrderSeries++
 			stats.OutOfOrderChunks += ooo
+			stats.OutOfOrderSeriesLabels = append(stats.OutOfOrderSeriesLabels, lset)
```

Comment:
We can just add a debug log here to print out the labels of every out-of-order series.
WDYT? @metonymic-smokey 
This seems simpler and we don't even need to save the slice.

Reply:
good point @yeya24 
I'll print it out here and remove this field from the struct. 
thanks!